src/c-writer.cc

```diff
@@ -656,6 +679,18 @@ std::string CWriter::ExportName(std::string_view module_name,
          MangleName(export_name);
 }
 
+/* The C symbol for a tail-callee export from this module. */
+std::string CWriter::TailCallExportName(std::string_view export_name) const {
+  return kTailCallSymbolPrefix + ExportName(export_name);
+}
+
+/* The C symbol for a tail-callee export from an arbitrary module. */
+// static
```

Comment:
Looks strange mixing C and C++ comments directly next each other like this.. maybe its fine since the `// static` is more like the type hint?

Reply:
Agreed that it's a little strange, but it matches the existing style (e.g. on CWriter::ExportName and CWriter::ModuleInstanceTypeName). We could maybe change all these together in a future PR?